pkgs/applications/display-managers/ly/default.nix

```diff
@@ -1,20 +1,26 @@
-{ stdenv, lib, fetchFromGitHub, linux-pam }:
+{ stdenv, lib, fetchFromGitHub, linux-pam, git, libxcb, ncurses }:
+
+# This package is intended to be reworked, it is temporarily using a fork since
+# the original developer used a non-standard method of specifying .gitmodules.
+# https://github.com/nullgemm/ly/pull/279
 
 stdenv.mkDerivation rec {
   pname = "ly";
-  version = "0.2.1";
+
+  hardeningDisable = [ "all" ];
```

Comment:
Can you be more selective and only disable the flags that fail?

Reply:
I tried, though I could not figure it out. This package is simply insecure, the intention as stated is to have @darthpjb rewrite this such that `hardeningDisable` is not required at all.

Reply:
Thats not great to hear.